src/mscorlib/src/System/Environment.cs

```diff
@@ -369,6 +369,9 @@ public static String CurrentDirectory
             [System.Security.SecuritySafeCritical]  // auto-generated
 #endif
             get {
+#if PLATFORM_UNIX
+                return Path.DirectorySeparatorCharAsString; // TODO: anything better to return?
```

Comment:
`/sbin` is a (still suboptimal) option: It is part of the Unix [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) and contains "essential system binaries, e.g., fsck, init, route.". Unfortunately, only _Linux_ distros fully implement the FHS; macOS does so only in part. (That's why `/lib`, arguably another candidate, wouldn't work - at least as a static value - because it doesn't  exist on macOS.